src/util/exception_utils.cpp

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************\
+
+Module: Exception helper utilities
+
+Author: Fotis Koutoulakis, fotis.koutoulakis@diffblue.com
+
+\*******************************************************************/
+
+#include "exception_utils.h"
+
+std::string invalid_user_input_exceptiont::what() const noexcept
+{
+  std::string res;
+  res += "\nInvalid User Input\n";
+  res += "Option: " + option + "\n";
+  res += "Reason: " + reason;
```

Comment:
Wishlist: please add a test for this, i.e., intentionally use CBMC with incorrect options.

Reply:
Do you need this as part of this PR, or can it go in as part of a follow up PR for example?

Reply:
Up to you, as long as it happens at some point :-)